routes/new_members.rb

```diff
@@ -50,6 +61,11 @@
 		redirect '/new_members/'
 	end
 
+	if !verify_recaptcha
+		flash(:alert, 'Google Recaptcha Verification failed', 'please try again.')
+		redirect '/new_members/'
```

Comment:
I would use this `redirect "new_members/sign_up/#{params[:event_id]}"` to redirect the user to the same page they were already navigated to instead of sending them back to `/new_members/`